### ISSUE TITLE
Submit button missing `aria-describedby`

### ISSUE THREAD

**Issue by wlabarron** [wlabarron/submit-show#41]
The final Submit button says "Uploading" while a show is uploading, with some help text underneath to explain the button activates once the show has uploaded. The help text should be linked to the button somehow. I think `aria-describedby` is the best attribute for this, but should double check before implementing.
